unittests/DxcSupport/WinAdapterTest.cpp

```diff
@@ -23,16 +23,7 @@ TEST(WArgVTest, suppressAndTrap) {
   std::wstring data [] = {L"a", L"b"};
   const wchar_t *ref_argv[] = {data[0].c_str(), data[1].c_str()};
   {
-    WArgV ArgV(argc, ref_argv);
-    const wchar_t **wargv = ArgV.argv();
-    for (int i=0;i<argc;++i) {
-      EXPECT_EQ(0, std::wcscmp(ref_argv[i], wargv[i]));
-    }
-  }
-  {
```

Comment:
I think we need to keep this around even if it's only run on Linux. This test was expressly added to test WArgV

Reply:
This removed part is testing the `wchar_t` overload of `WArgV` constructor, which is getting removed. The test code below tests the remaining `char` overload of the constructor.